spec/project_printer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "spec_helper"
+require "project_printer"
+
+RSpec.describe ProjectPrinter do
+  describe "#changed_name" do
+    it "changes the project name" do
+      stdout = StringIO.new("")
+      project_name = "First project"
+      new_project_name = "Updated project"
+      project_printer = ProjectPrinter.new(stdout)
+      output = "\e[38;5;40mChanged project name from\e[0m 'First project' \e[38;5;40mto\e[0m 'Updated project'"
+
+      project_printer.changed_name(project_name, new_project_name)
+
+      expect(stdout.string).to include(output)
+    end
+  end
+
+  describe "#created" do
+    it "creates the project name" do
+      stdout = StringIO.new("")
+      project_name = "First project"
+      project_printer = ProjectPrinter.new(stdout)
+      output = "\e[38;5;40mCreated project:\e[0m 'First project'"
+
+      project_printer.created(project_name)
+
+      expect(stdout.string).to include(output)
```

Comment:
Same feedback about escape chars and output assertion being readable. Throughout!